nservicebus/sqlserver/design.md

```diff
@@ -49,6 +49,10 @@ CREATE TABLE [schema].[queuename](
 CREATE CLUSTERED INDEX [Index_RowVersion] ON [schema].[queuename](
 	[RowVersion] ASC
 ) WITH (PAD_INDEX  = OFF, STATISTICS_NORECOMPUTE  = OFF, SORT_IN_TEMPDB = OFF, IGNORE_DUP_KEY = OFF, DROP_EXISTING = OFF, ONLINE = OFF, ALLOW_ROW_LOCKS  = ON, ALLOW_PAGE_LOCKS  = ON) ON [PRIMARY]
+
+CREATE NONCLUSTERED INDEX [Index_Expires] ON [schema].[queuename](
```

Comment:
I think we need to specify differences between version. Index should not exists for V1 for example